server/repositories/cmsApi.js

```diff
@@ -53,7 +53,7 @@ class CmsApi {
     try {
       return await call();
     } catch (err) {
-      if (err.response && statusCodes.includes(err.response.status)) {
+      if (statusCodes.includes(err?.response?.status)) {
```

Comment:
Fixed the commit merge to take your change. 
Only nitpicking comment is if err doesn't exist then the next line would blow up. 
Err should always be present according to the docs so NBD